app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -266,7 +266,11 @@ def join
   end
 
   def get_recordings
-    params.require(:recordID) if Rails.configuration.x.get_recordings_api_filtered
+    if Rails.configuration.x.get_recordings_api_filtered
+      if params[:recordID].blank? && params[:meetingID].blank?
+        raise BBBError.new('recordID/meetingID is missing', 'You must specify a recordID/meetingID.')
```

Comment:
The error message should be impersonal

`param meetingID or recordID must be included`